### DISCUSSION TITLE
Which project and example to follow?

### DISCUSSION THREAD

**Discussion by carmenchapa** [storybookjs/react-native#407] [Q&A] [answered]
Hey!
Thanks a lot for all the work on this.
I am using React Native + Expo + Storybook 5.3 and would like to update to Storybook 6.x for react-native-web usage.
I am a bit confused about this two projects, and which is the good one:
[storybook-react-native in Gitlab](https://gitlab.com/risserlabs/community/storybook-react-native/)
[storybook/react-native in Github](https://github.com/storybookjs/react-native)

Also, I would like to use [this starter](https://github.com/dannyhw/expo-storybook-starter) as an example, but I can see there is [this other one] too(https://github.com/dannyhw/expo-template-storybook) 
Is there a reason to use one or another? 
tagging @dannyhw
thanks!

**Comment by dannyhw** [comment, marked as answer]
The template is used with the expo cli to generate projects like this `expo init --template expo-template-storybook AwesomeStorybook` so isn't an example project itself.

The starter project is something you can copy.

Storybook is on github and anything on gitlab is a separate copy/fork created by someone else.

**Reply by carmenchapa** [reply]
Thank you for your quick answer 🙏
